ql/src/java/org/apache/hadoop/hive/ql/metadata/Table.java

```
@@ -308,7 +309,13 @@ public HiveStorageHandler getStorageHandler() {
             return null;
           }
           inputFormatClass = getStorageHandler().getInputFormatClass();
+          if ("com.twitter.elephantbird.mapred.input.HiveMultiInputFormat".equals(inputFormatClass.getName())) {
```

I think we won't come into this branch. Will we?

---

Yes I was reading about storage handler recently. Right now we do not use Storage Handlers with HIVE. Most likely, we will not come. Do you want to keep the check as a safeguard or let it fail if ever and fix it at that point